scripts/pash-nginx.conf

```diff
@@ -74,6 +90,25 @@ server {
   }
 }
 
+server {
+  listen 80;
+  server_name perf-ctrl.pash.ndr.md;
```

Comment:
Not for now, but for later:

The `ctrl.pash.ndr.md` could remain the same—pointing to the same controller script—but just have a few additional endpoints for performance (and locks). Generally having a single controller is ideal, because we can handle all queuing / concurrency / barrier logic at a single place.